figlet.yaml

```diff
@@ -0,0 +1,53 @@
+#nolint:valid-pipeline-git-checkout-tag
+package:
+  name: figlet
+  version: 2.2.5
+  epoch: 0
+  description: Program that creates large characters out of ordinary screen characters.
+  copyright:
+    - license: BSD-3-Clause
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/cmatsuoka/figlet
+      branch: master
+      expected-commit: 202a0a8110650a943f1125f536b3bb455cf72ee1
+
+  - name: Configure and build
+    runs: |
+      make clean
```

Comment:
Is it possible to use `- uses: autoconf/make` and `- uses: autoconf/make-install` pipelines here?

Reply:
Updated the step using `- uses: autoconf/make-install` step.